unlock-protocol.com/src/components/interface/Membership.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import styled from 'styled-components'
+
+export class Membership extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isMember: 'pending',
+    }
+  }
+
+  componentDidMount() {
+    // Connect to Unlock and get the state!
+    window.addEventListener('unlockProtocol', event => {
+      this.setState(state => ({
+        ...state,
+        isMember: event.detail === 'unlocked' ? 'yes' : 'no',
```

Comment:
We will expand the messages emitted in order to support the tickets app to include emitting the data needed to generate the QR code. The isMember check should explicitly check for `'locked'`